scripts/install-multi-user.sh

```diff
@@ -704,5 +843,15 @@ main() {
     trap finish_success EXIT
 }
 
-
-main
+# ACTION for override
+case "${1-}" in
+    # uninstall)
+    #     shift
+    #     uninstall "$@";;
+    ""|install|*)
+        if ! [ "${#@}" = 0 ]; then
+            shift
+        fi
+        # same as the no-arg condition for now (but, explicit)
+        main "$@";;
+esac
```

Comment:
```suggestion
    install|*)
      if ! [ "${#@}" = 0 ]; then
        shift
      fi
      # same as the no-arg condition for now (but, explicit)
      main "$@"
      ;;
esac
```

I would do it like this.

Reply:
You're correct, but I'm under-explaining myself here (and you are picking at something that _is_ weird!)

I've updated this locally to hopefully clarify.